app_flutter/lib/widgets/status_grid.dart

```diff
@@ -5,13 +5,15 @@
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 
```

Comment:
With TaskMatrix gone, it might make more sense to call this TaskGrid. That would make the files a bit more cohesive for the build dashboard widgets

Reply:
done